skt/reporter.py

```diff
@@ -343,24 +343,28 @@ def getjobresults(self):
 
```

Comment:
Comment for the whole commit:

This commit contains at least two logical changes: fixing extracting wrong console logs, and adding extracting console logs from other runs. Following the rule "one logical change per commit" this one should be at least two commits. That way, if e.g. it turns out that we don't need logs from all runs after all, it would be easy to revert it, or e.g. if we had to fix wrong console logs in production urgently, it would be easy to cherry-pick it there.